python/modem-firmware-1.3+/device_credentials_installer.py

```diff
@@ -581,8 +582,8 @@ def main():
         attr = ',\"CN={}\"'.format(custom_dev_id)
 
     write_line('AT%KEYGEN={},2,0{}'.format(args.sectag,attr))
-    # include the CRLF in OK because 'OK' could be found in the CSR string
-    retval, output = wait_for_prompt(b'OK\r\n', b'ERROR', store=b'%KEYGEN:')
+    # include the CR in OK because 'OK' could be found in the CSR string
```

Comment:
Does this change fix the CR issue you mentioned in Slack?

Reply:
And, does it work on Windows still?

Reply:
yes.

i don't think this should matter, since it is now doing a less-strict check.
but for the serial port change, could you please test on windows?

Reply:
Yes, just tried it and it finds my 9161DK.